src/main/fc/fc_core.c

```diff
@@ -129,6 +130,7 @@ int16_t magHold;
 #endif
 
 static bool flipOverAfterCrashMode = false;
+static bool constantIdleMode = false;
```

Comment:
This is looking good now. One possible improvement I can see: Instead of having separate bools for flip over after crash and constant idle, why not combine them into an enum `mixerMode` or suchlike, with values of `NORMAL`, `FLIP_OVER_AFTER_CRASH`, and `CONSTANT_IDLE`? Only one of the three can be active at any given time, and this will simplify the rest of the code.

Reply:
I suggest this should be taken care of in a separate issue, since it also affects "flip over after crash" and could be / have been implemented even if there was no "constant idle" mode ;)
Agree?

Reply:
I suggest this should be done properly the first time round, not done in a badly designed way in the hope that it will be fixed later on.

Reply:
I'm totally with you here, just wanted to avoid implementing stuff that's not per se related to the feature implementation itself, since, technically speaking, it's indepentent ;)
Hope you get my point, i'll solve it by implementing a mixerMode enum right here in the PR as you described. It totally makes sense, and might/will come in handy for future stuff, i'm sure..